scene/2d/box2d_world.cpp

```diff
@@ -712,6 +712,9 @@ void Box2DWorld::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("cast_motion", "query"), &Box2DWorld::cast_motion);
 	//ClassDB::bind_method(D_METHOD("collide_shape", "shape", "max_results"), &PhysicsDirectSpaceState2D::_collide_shape, DEFVAL(32));
 
+	ClassDB::bind_method(D_METHOD("intersect_point_fast", "output", "point", "max_results", "exclude", "collision_mask", "collide_with_bodies", "collide_with_sensors", "collision_layer", "group"), &Box2DWorld::intersect_point_fast, DEFVAL(32), DEFVAL(Array()), DEFVAL(0xFFFFFFFF), DEFVAL(true), DEFVAL(false), DEFVAL(0x0), DEFVAL(0));
```

Comment:
Arent the default values 32 and Array() backwards? The default values dont seem to align with the parameters?

Reply:
They look aligned to me. They align starting at the end of the parameter list

Reply:
I see